mikeio/dfs.py

```diff
@@ -13,6 +13,7 @@
 )
 from .eum import ItemInfo, TimeStepUnit, EUMType, EUMUnit
 from .custom_exceptions import DataDimensionMismatch, ItemNumbersError
+from .dfsutil import valid_item_numbers, valid_timesteps, get_item_info
```

Comment:
Rename dfsutil to _dfsutil to indicate it is for internal use?

Reply:
Maybe. But perhaps **better to rename the methods to underscore** _variants()? I looked at this https://stackoverflow.com/questions/3602110/python-private-module-in-a-package and https://stackoverflow.com/questions/55157525/leading-underscore-before-the-name-of-python-module  "In the Python standard library, a leading underscore in a module name may denote an accelerator module: a module written in C that provides a more performant implementation of the pure Python version" https://www.python.org/dev/peps/pep-0399/#details

Reply:
agreed to rename methods instead of module (on phone)